src/s2python/authorization/client.py

```diff
@@ -1,67 +1,361 @@
-from abc import ABC, abstractmethod
-from typing import Any, Dict
+"""
+S2 protocol client for handling pairing and secure connections.
+"""
 
+import abc
```

Comment:
this is a legitimate warning, but in our case it's unavoidable, so I suggest just switching it off locally.